### PR TITLE
[Lighthouse] Style and link updates

### PR DESCRIPTION
**Pull request by AaronForinton**
- Removed instances of i.e. and e.g.
- Removed as many developers.google.com and html5rocks URLs as possible, particularly for the Workbox and DevTools docs the redirects weren't working
- Caught a few minor typos and suggested a few minor enhancements
- Updated browser compatibility in a couple of spots (and dated those pages)

### CODE REVIEW

File: site/en/docs/lighthouse/overview/index.md [hunk 1]
```diff
@@ -7,13 +7,15 @@ date: 2016-09-27
...
 
 <figure>
   {% Img src="image/MtjnObpuceYe3ijODN3a79WrxLU2/QXJ5lmcjHEFLTHg5B4o8.svg", class="float-right", alt="Lighthouse logo", width="150", height="150" %}
 </figure>
 
-You can run Lighthouse in Chrome DevTools, from the command line, or as a Node module. You give Lighthouse a URL to audit, it runs a series of audits against the page, and then it generates a report on how well the page did. From there, use the failing audits as indicators on how to improve the page. Each audit has a reference doc explaining why the audit is important, as well as how to fix it.
+You can run Lighthouse in Chrome DevTools, from the command line, or as a Node module. You give Lighthouse a URL to audit, it runs a series of audits against the page,
+and then it generates a report on how well the page did. From there, use the failing audits as indicators on how to improve the page. Each audit has a reference doc explaining why
+the audit is important, as well as how to fix it.
```
Comment (AaronForinton) [inline site/en/docs/lighthouse/overview/index.md:18]: Looks ugly here splitting this line makes the page more navigable: 

[https://raw.githubusercontent.com/GoogleChrome/developer.chrome.com/f4eb081d346113aa2baa92628108c8baa8aba90b/site/en/docs/lighthouse/overview/index.md](https://raw.githubusercontent.com/GoogleChrome/developer.chrome.com/f4eb081d346113aa2baa92628108c8baa8aba90b/site/en/docs/lighthouse/overview/index.md)

File: site/en/docs/lighthouse/best-practices/image-aspect-ratio/index.md [hunk 2]
```diff
@@ -8,14 +8,14 @@ updated: 2020-04-29
...
+to the aspect ratio in its source file (the _natural_ aspect ratio),
 the rendered image may look distorted,
 possibly creating an unpleasant user experience.
 
 ## How the Lighthouse image aspect ratio audit fails
 
-[Lighthouse](https://developers.google.com/web/tools/lighthouse/) flags any
-image with a rendered dimension more than a few pixels different from the expected dimension
+[Lighthouse](/docs/lighthouse/overview/) flags any
+image with a rendered dimension of more than a few pixels difference to the expected dimension
```
Comment (AaronForinton) [inline site/en/docs/lighthouse/best-practices/image-aspect-ratio/index.md:18]: Defer to others on whether I'm actually right for this one or not

File: site/en/docs/lighthouse/best-practices/image-aspect-ratio/index.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ when rendered at its natural ratio:
 
 There are two common causes for an incorrect image aspect ratio:
 
-- An image is set to explicit width and height values that differ from the source image's dimensions.
+- An image is set with explicit width and height values that differ from the source image's dimensions.
```
Comment (AaronForinton) [inline site/en/docs/lighthouse/best-practices/image-aspect-ratio/index.md:27]:
```suggestion
- An image is set to explicit width and height dimensions that differ from the source image's dimensions.
```

Alternately (keeps the 'to')

**Comment by AaronForinton** [comment]
@sofiayem I wanted to trawl through all the Lighthouse docs after our discussion the other day. While I was doing that I decided I'd change a bunch of URLs and I've suggested a few small changes to sentences here and there too

**Review by rachelandrew — APPROVED** [review]

**Review by sofiayem — APPROVED** [review]

**Pull request merged**